calibrators/marker_radar_lidar_calibrator/README.md

```diff
@@ -0,0 +1,141 @@
+# marker_radar_lidar_calibrator
+
+A tutorial for this calibrator can be found [here](../docs/tutorials/marker_radar_lidar_calibrator.md)
+
+## Purpose
+
+The package `marker_radar_lidar_calibrator` allows extrinsic calibration between radar and 3d lidar sensors used in autonomous driving and robotics.
+
+Currently, the calibrator only supports the radar that includes distance and azimuth angle, but without elevation angle. For example, ARS 408 radar can be calibrated with this tool. Also, note that the 3d lidar should have a resolution that is high enough to scan several points on the radar reflector (calibration target).
```

Comment:
"supports the radar" is wrong

Reply:
At this point, there has bee no mention to the corner reflector. Please add a link to the corresponding section

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/e56ecbf52c8aa53f6d31529516aa4a7987a71767

Reply:
`supports the radars` -> `supports radars`

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/cb86959d813939afe33ba5d7ac34793da932546e